.stylelintrc

```diff
@@ -1,29 +1,3 @@
 {
-  "extends": "stylelint-config-sass-guidelines",
-  "ignoreFiles": [
```

Comment:
We should keep the ignored files.  Sorry that was my fault stating you could remove the entire stylelint config 😱